.github/workflows/build.yml

```diff
@@ -23,62 +23,29 @@ jobs:
           - "4.11"
           - "4.12"
           - "4.13"
+          - "4.14"
           - "5.0"
           - "5.1"
-        skip-test:
```

Comment:
I find the previous version much easier to understand.

Reply:
This PR kind of inline the logic previously encoded by skip-* parameters. We know have to go over all step understand what's done for what version. I think this is going in the wrong direction.